site/en/docs/extensions/mv3/getstarted/development-basics/index.md

```diff
@@ -0,0 +1,271 @@
+3. Click the **Load unpacked** button and select the extension directory.
+    <figure>
+    {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/BzVElZpUtNE4dueVPSp3.png", alt="Chrome Extension Management page", 
+    width="400", height="183",  class="screenshot" %}
+
+      <figcaption>
+      Chrome Management (chrome://extensions)
+      </figcaption>
+    </figure>
+
+Ta-da! The extension has been successfully installed. Because no extension icons were included in
+the manifest, a generic icon will be created for the extension.
+
+## Pinning the extension {: #pin }
+
+By default, when you load your extension locally, it will appear in the **extensions menu** {% Img
+src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/f5i7AgfauCfoQJxnn3kU.png", alt="Puzzle", width="24",
+height="24" %}. Pin your extension to the toolbar to quickly access your extension during
+development.
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/LahIugYHQW3QpHM0z1qZ.png", 
+alt="Pinning the extension", width="358", height="248", class="screenshot" %}
+  <figcaption>
+  Pinning the extension
+  </figcaption>
+</figure>
+
+Click on the extension’s action (toolbar icon); you should see a popup.
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/xjKRmWMgwMm7Kdf72bkj.png", 
+alt="hello world extension", width="206", height="130", class="screenshot" %}
+  <figcaption>
+  Hello World extension
+  </figcaption>
+</figure>
+
+## Reloading the extension {: #reload }
+
+Let’s go back to the code and change the content of the `H1` element in your HTML file.
+
+{% Label %}hello.html{% endLabel %}
+
+```html
+<html>
+ <body>
+   <h1>Hello Extensions of the world! 👋</h1>
+ </body>
+</html>
+```
+
+After saving the file, go to the Extension Management page. You can see the changes made by clicking
+on the refresh icon next to the **on/off** toggle:
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/QNtwZICbwsGO3gacCJEd.png", 
+alt="Reload an extension", width="500", height="233", class="screenshot" %}
+
+{% Aside %}
+
+💡 **TIP**: Remember to refresh your extension every time you make changes.
+
+{% endAside %}
+
+## Finding console logs and errors {: #logs }
+
+### Console logs {: #console}
+
+During development, you can debug your code by accessing the browser console logs. In this case, we
+will locate the logs for the popup. Start by adding a script tag to `hello.html`.
+
+{% Label %}hello.html{% endLabel %}
+
+```html
+<html>
+ <head>
+   </head>
+ <body>
+   <h1>Hello Extensions of the world! 👋/h1>
+   <script src="popup.js"></script>
+ </body>
+</html>
+```
+
+Create a `popup.js` file and add the following code to output the log to the console:
+
+{% Label %}popup.js{% endLabel %}
+
+```js
+console.log("This is a popup!")
+```
+
+To see this message logged in the Console, complete the following steps:
+
+  1. Refresh the extension
+  2. Open the popup
+  3. Right-click on the popup
+  4. Select **Inspect**. 
```

Comment:
As a general nit, please review period use in unordered and ordered lists for consistency. I don't have a strong preference on whether list items should or should not end with periods, but we should be more consistent about what pattern we use and we should definitely stay consistent within a single list.

For example, items 4 and 5 are the only items in this list that end with periods.

Reply:
Um yes, I missed this 😳... I think I rewrote this list a few times and failed to capture those elusive periods... Thanks for catching this! Adding this to my next commit

Reply:
Ask me about this some time. The actual rule on this is more complicated.